pkg/virt-handler/vm.go

```diff
@@ -2839,6 +2839,8 @@ func (d *VirtualMachineController) vmUpdateHelperMigrationTarget(origVMI *v1.Vir
 		}
 		log.Log.Object(vmi).Infof("Signaled target pod for failed migration to clean up")
 		// nothing left to do here if the migration failed.
+		// Re-enqueue to trigger handler final cleanup
+		d.queue.AddAfter(controller.VirtualMachineInstanceKey(vmi), time.Second)
```

Comment:
I see one more call to `SignalTargetPodCleanup`. Also can we get a unit test at least?

Reply:
I think this could happen. Do you think we need to increase the time of the addAfter?

Reply:
Extended the already existing test. Is that ok?